changelogs/unreleased/issue-fix-core-build-failure.yml

```diff
@@ -0,0 +1,5 @@
+change-type: patch
+description: Fix core build failure
+destination-branches:
+- master
```

Comment:
Shouldn't this be all branches?

Reply:
It only made it to master

Reply:
This [PR](https://github.com/inmanta/inmanta-core/pull/7513) I mean only made it to master and got rejected on iso7 and iso6 because of failing tests

Reply:
You'll probably need to delete/update its change entry then, or merge dummy ones into the other branches